CHANGELOG.md

```diff
@@ -1,6 +1,50 @@
 # Changelog
 
 
+## [1.52](https://github.com/brave/brave-ios/releases/tag/v1.52)
+
+### Web3
+
+- Added support for transactions from all networks on activity tab. ([#7233](https://github.com/brave/brave-ios/issues/7233))
+- Added support for Solana v0 VersionedTransactions. ([#7211](https://github.com/brave/brave-ios/issues/7211))
+- Implemented NFT auto discovery in Wallet. ([#7370](https://github.com/brave/brave-ios/issues/7370))
+- Implemented selected network per origin. ([#7225](https://github.com/brave/brave-ios/issues/7225))
```

Comment:
Please update to `Added the ability to connect to DApps by selected network per origin.` to keep it consistent with desktop notes

Reply:
SG, thanks.